src/modules/modals/TransferTable.vue

```diff
@@ -2,49 +2,56 @@
 	<NcModal v-if="showModal"
 		size="normal"
 		@close="actionCancel">
-		<div class="modal__content">
+		<div class="modal__content" data-cy="transferTableModal">
 			<div class="row">
 				<div class="col-4">
-					<h2>{{ t('tables', 'Transfer tables ownership') }}</h2>
+					<h2>{{ t('tables', 'Transfer table') }}</h2>
 				</div>
 			</div>
 			<div class="row">
-				// TODO: add actual owner
+				<div class="col-4 mandatory space-T">
+					{{ t('tables', 'Owner') }}
+				</div>
+
+				<!-- We don't need to show this, since the current user will only see this if they're the owner? -->
+				<div v-if="localTable !== undefined" class="col-3 inline space-T-small">
+					<NcUserBubble
+						:margin="4"
+						:size="30"
+						:display-name="localTable.ownerDisplayName"
+						:user="localTable.owner" />
+				</div>
 			</div>
 			<div class="row">
-				<NcUserAndGroupPicker :users="true" :groups="false" :value.sync="newOwnerUserId" />
+				<NcUserAndGroupPicker :select-users="true" :select-groups="false" :new-owner-user-id.sync="newOwnerUserId" />
 			</div>
 			<div class="row">
 				<div class="fix-col-4 space-T justify-between">
-					<NcButton v-if="!prepareTransferTable" type="error" @click="prepareTransferTable = true">
+					<NcButton type="warning" :disabled="newOwnerUserId === ''" @click="transferMe">
 						{{ t('tables', 'Transfer') }}
 					</NcButton>
-					<NcButton v-if="prepareTransferTable"
-						:wide="true"
-						type="error"
-						@click="transferMe">
-						{{ t('tables', 'I really want to transfer this table!') }}
-					</NcButton>
```

Comment:
Not sure if we want to keep this and make the user confirm when transfering a table. I read  that it's good UX to have such a confirmation when a user is doing a destructive or irreversible action, which maybe applies here?